src/main/java/be/syntra/devshop/DevshopFront/services/CartServiceImpl.java

```diff
@@ -139,11 +138,17 @@ public BigDecimal getCartTotalPrice(CartDto currentCart) {
     @Override
     public CartDisplayDto getCartDisplayDto() {
         return CartDisplayDto.builder()
-                .cartProductDtoSet(getProducts())
+                .cartProductDtoSet(getSortedProducts())
                 .cartProductsIdSet(getProductIds())
                 .build();
     }
 
+    private Set<CartProductDto> getSortedProducts() {
+        SortedSet<CartProductDto> sorted = new TreeSet<>(Comparator.comparing(c -> c.getProductDto().getName()));
```

Comment:
one letter abbriviations are only acceptable in a basic for-loop. To make this method future proof for other programmers who might change some things, it's better to get accustomed to using full names like 'cartProductDto -> cartProductDto.getProductDto().getName()